### DISCUSSION TITLE
Feature Request: Dep Graph embedded in VSCode

### DISCUSSION THREAD

**Discussion by jeffbcross** [nrwl/nx-console#1021] [Feature Requests]
Currently, when I click "Dep Graph" then "execute", it opens a new browser tab/window outside of VSCode. I'd prefer if it opened inside VSCode.

**Comment by sandikbarr** [comment]
We should be able to show the dep-graph in a tab in VS Code with the built in Simple Browser command 
```
commands.executeCommand('simpleBrowser.show', 'http://localhost:4211/');
```
![Screen Shot 2021-04-05 at 2 51 03 PM](https://user-images.githubusercontent.com/2250413/113619269-673a6e80-961e-11eb-8424-e3f8e825ec0f.png)

It's the `nx dep-graph` command itself that opens the default browser (if you do not use the `--file=output` option). The dep-graph command in Nx Console is just running Nx commands with the selected options. So it seems like we also would want an additional option to not have `nx dep-graph` open the browser when it's started.

Another way of implementing this might be to use the `nx dep-graph --file=output.html` option and show the dep-graph output.html in a VS Code web view. This web view way doesn't really seem to fit within the "Common Nx Commands" panel as the intention there is to build up `nx` commands to be executed.
